src/super_gradients/training/datasets/detection_datasets/pascal_voc_detection.py

```diff
@@ -132,3 +140,29 @@ def get_coord(box_coord):
                 new_label_path = (dest_labels_path / img_path.name).with_suffix('.txt')
                 img_path.rename(new_img_path)  # Move image to dest folder
                 _parse_and_save_labels(data_path, new_label_path, year, id)
+
+
+class PascalVOCUnifiedDetectionTrainDataset(ConcatDataset):
```

Comment:
where is the code of the previous version (before using yaml). I expected to see many lines being deleted

Reply:
This is because the main has already been done in https://github.com/Deci-AI/super-gradients/pull/278 .

I simply added the basic functionality of unifying the datasets (most common way to train on voc) as unified trainset (this was hard coded in the dataset interface).